src/Places/Suggestions/mockSuggestions.ts

```diff
@@ -0,0 +1,41 @@
+import { PlacesSuggestionsResponse } from '../../types'
+
+export const mockPlacesSuggestionsResponse: PlacesSuggestionsResponse = {
```

Comment:
A bit nitpick: Just based on the name ending with "Response" I'd expect this to be the full response including `data` as an attribute or at leat being the array of Suggestions. Wdyt? Really, this is just `mockSuggestion`, no?

Reply:
Also, why not use the makeMock pattern here?

```ts
import { DuffelAPI } from "@lib/types"

let seedId = 1
const defaultObject: DuffelAPI.Types.TYPE = { 
  id: `pre_${(++seedId).toString().padStart(5, '0')}`
}

export const makeMock__THING__ = (
  extendDefault?: Partial<DuffelAPI.Types.TYPE>
): DuffelAPI.Types.TYPE => Object.assign({}, defaultObject, extendDefault || {})

```

Reply:
This is a pattern we don't have inside `duffel-api-javascript` and I think it would increase the knowledge curve for someone that would like to contribute to the repository! I'm just following.

Regarding the name convection, I will update it to match our other mocks!

Reply:
The reason might be [this](https://github.com/duffelhq/duffel-api-javascript/pull/569#discussion_r922240592) lol (the fact we don't have that pattern in this code base). But still curious in case there's a different reason there.

Reply:
Happy to take this conversation to Slack, when we started the project we never thought about adding `make-mock` functions, we tried as much we could to keep the repository simple.

Reply:
all good, just thought this was dashboard. Thank you!